scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/tsop.yaml

```diff
@@ -0,0 +1,26 @@
+FileHeader:
+  library_Suffix: 'SO'
+  device_type: 'TSOP'
+
+TSOP-6_1.50mmx2.90mm_P0.95mm:
+  size_source: 'https://www.nxp.com/packages/SOT457'
+  body_size_x:
+    minimum: 1.3
+    maximum: 1.7
+  body_size_y:
+    minimum: 2.7
+    maximum: 3.1
+  overall_size_x:
+    minimum: 2.5
+    maximum: 3.0
+  overall_height:
+    minimum: 0.9
+    maximum: 1.1
+  lead_width:
+    minimum: 0.25
+    maximum: 0.40
+  lead_len:
+    nominal: 0.58
```

Comment:
only minimum and maximum are given in the datasheet (dimension LP)

For reference: 0.2 ... 0.6

Reply:
Both issues address now. Thanks!